client/src/components/DashboardDropdown/DashboardDropdown.scss

```diff
@@ -0,0 +1,78 @@
+@import '../../scssStyles/mixins';
+
+.dashboard-dropdown-bg {
+  top: 0;
+  height: 100%;
+
+  width: fit-content;
+}
+
+.dashboard-dropdown-bg-display {
+  opacity: 1;
+}
+
+.dashboard-dropdown-bg-hide {
+  pointer-events: none;
+  opacity: 0;
+}
+
+.dashboard-dropdown-container {
+  position: absolute;
+  display: flex;
+  flex-direction: column;
+  border-radius: 6px;
+  width: max(130px, 13%);
```

Comment:
just a note on the width... i'm trying to figure out a way where you can set the width of the dropdown container according to the width of the "dropdown part of it" since this is fixed, while the other higher dropdown part is like a flex box...

ideally it would be best if the dropdown image and the actual dropdown clicking container were in the same "div", so that way the widths would be the same... but overall it's ok for now, there's quite a bit of refractoring that's needed so we can come back to this later!

(hope that kinda makes sense btw, too many instances of dropdown ahha)